tools/AzDev/src/Services/Conventions.cs

```diff
@@ -107,11 +107,13 @@ internal static bool IsExcludedProjectDirectory(IFileSystem fs, string path, out
                 return true;
             }
 
-            if (!TryGetOnlyCsprojPath(fs, path, out var _, out var cannotFindCsproj))
+            if (!TryGetOnlyCsprojPath(fs, path, out var _, out var cannotFindCsproj)
+                && !IsAutorestBasedProject(path, out var notAutorestBased))
```

Comment:
Change to core business logic is here. Autorest-based projects as well as sdk-based ones both have a .csproj file that can be used to identify it is a project. This no longer holds for autorest-based projects because their csprojs are moved to another folder.